lib/trento_web/openapi/api_spec.ex

```diff
@@ -139,34 +144,46 @@ defmodule TrentoWeb.OpenApi.ApiSpec do
           }
         end
       end
+    end
+  end
 
-      defp build_paths_for_version(version, router) do
-        available_versions = router.available_api_versions()
+  def build_paths_for_version("latest", router) do
+    router
+    |> Paths.from_router()
+    |> Enum.filter(fn {path, path_item} ->
+      path_item
+      |> Map.from_struct()
+      |> Map.values()
+      |> Enum.any?(fn
+        %Operation{deprecated: true} -> false
```

Comment:
question/comment: I guess this is filtering out those operations having a `deprecated: true`.
I see we're deprecating both operations and schemas. Do you know if the linter is preventing us from either of these situations: 1) deprecated operation with non-deprecated schema (and not used elsewhere) or 2) non-deprecated operation with at least one response schema deprecated.

I don't think so, though, it's going to be just us being cautious and careful when we deprecate something.

Reply:
> 1) deprecated operation with non-deprecated schema (and not used elsewhere)